### DISCUSSION TITLE
H1 tag

### DISCUSSION THREAD

**Discussion by Nwadkoko** [CaiJimmy/hugo-theme-stack#640] [Ideas]
Hi everyone,

Can we change the H1 tag for SEO purposes? Currently, each H1 tag on each page is the site name.

**Comment by kininblog** [comment]
Hi, 
to solve this problem in hugo stack theme, adding conditionals did not work at all.

so I duplicated the header.html and details.html files, the solution for the[ h1 tag hugo stack theme ](https://akazed.com/solusi-tag-h1-hugo-stack-theme/)I have written more completely on the akazed blog
